classes/submission/Submission.inc.php

```diff
@@ -233,14 +233,6 @@ function setLicenseURL($licenseURL) {
 	}
 
 	/**
-	 * Get comments to editor.
-	 * @return string
-	 */
-	function getCommentsToEditor() {
```

Comment:
This function is called on a few other places as well, but I can do that once this PR is merged:
s. `classes/article/ArticleDAO.inc.php` and `lib/pkp/plugins/importexport/native/filter/SubmissionNativeXmlFilter.inc.php`